src/config.h

```diff
@@ -74,6 +74,7 @@ struct Config {
     std::vector<NamedString> config_entries;  // Entries go into OrtSessionOptions::AddConfigEntry
 
     std::vector<ProviderOptions> provider_options;
+    std::vector<std::string> providers;
```

Comment:
```suggestion
    std::vector<std::string> providers;  // List of providers to use at runtime, not persisted in the json currently
```
This comment makes my current understanding more explicit